pkg/oauth/server.go

```diff
@@ -223,18 +223,10 @@ func (s *server) RegisterRoutes(server *web.Server) {
 
 	csrfMiddleware := middleware.CSRF("_csrf", "/", s.config.CSRFAuthKey)
 
-	api := root.Group("/api", csrfMiddleware)
-	api.POST("/auth/login", s.Login)
-	api.POST("/auth/logout", s.Logout, s.requireLogin)
-	api.GET("/me", s.CurrentUser, s.requireLogin)
-
 	page := root.Group("", csrfMiddleware)
-	page.GET("/login", webui.Template.Handler, s.redirectToNext)
 	page.GET("/logout", s.ClientLogout)
```

Comment:
Maybe add a note that this route is in use by existing OAuth clients, meaning that it's public API and should therefore not be removed in v3.